incubator/fluentd-cloudwatch/templates/daemonset.yaml

```
@@ -32,11 +32,11 @@ spec:
               mountPath: /config-volume
             - name: config
               mountPath: /etc/fluentd
+      hostNetwork: {{ default false .Values.hostNetwork }}
```

No need for a default here, since - like the other `.Values` declarations in this template - the line in `values.yaml` provides the default.

```suggestion
      hostNetwork: {{ .Values.hostNetwork }}
```

---

Modified.